motion_predict/src/motion_predict.cpp

```diff
@@ -66,7 +66,7 @@ carma_perception_msgs::msg::PredictedState predictState(const geometry_msgs::msg
 
   pobj.predicted_velocity.angular.x=twist.angular.x;
   pobj.predicted_velocity.angular.y=twist.angular.y;
-  pobj.predicted_velocity.angular.z=twist.angular.y;
+  pobj.predicted_velocity.angular.z=twist.angular.z;
```

Comment:
Can we update the `testpredictState` unit test to test this change as well?

Reply:
updated